packages/database/src/core/migrations/20250410213322-DatatrakWebInitialSchema-modifies-schema.js

```diff
@@ -0,0 +1,383 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.executeSql(`
```

Comment:
These are all the schemas that I can think of that we need for datatrak web. All related to the sets below:
- entity set
- question
- survey
- project

If there's any missing, we can add them later easily